### DISCUSSION TITLE
Nulleinspeisung warum?

### DISCUSSION THREAD

**Discussion by D3R-ST3FAN** [tbnobody/OpenDTU#327] [General]
Ich lese hier ja total oft Fragen rund um Nulleinspeisung.
Hier scheinen einige ja extrem viel Aufwand zu betreiben.

Warum? Ist ernst gemeint, ich verstehe es nicht.

Zum 1.1. wird ja auch die 70% Regelung wegfallen. 
Und der HM-1500 dürft der größte WR sein, den man mit OpenDTU betreiben kann --> also ist ja auch die Anlagengröße begrenzt.

Helft mir...

**Comment by MaxDaNr1** [comment]
Null-Einspeisung ist für viele interessant, die eine kleine PV-Anlage unangemeldet betreiben wollen. Anlagen müssen beim Netzbetreiber und beim Markstammdatenregister angemeldet werden. Das ist sehr bürokratisch und manche Netzbetreiber fordern sogar den Einbau von Wieland-/Einspeise-Steckdosen. Wenn man einen alten Ferraris-Zähler ohne Rücklaufsperre hat, dann kann es auch sein, dass vor Inbetriebnahme der Mini-PV der Zähler getauscht werden muss etc.
Insgesamt spart man sich also Ärger und Aufwand, wenn man den PV-Strom wirklich nur für sich selbst nutzt und das geht nur mit Nulleinspeisung.

**Comment by battifreund** [comment]
Ich finde, dass der bürokratische Aufwand beim Betrieb eines regulären BKW (bis 600Wip) jetzt im Verhältnis zum Aufwand der Einrichtung einer 0-Einspeisung deutlich geringer ist. Selbst wenn der Zähler getauscht werden muss, ist alles i.d.R. in maximal 30 Minuten Eigenaufwand erledigt. Wie lange der NB benötigt, um den Zähler zu tauschen, steht auf einem anderem Blatt.

Das Einrichten einer 0-Einspeisung kann im derzeitigen Entwicklungsstadium kaum in 30 Minuten erledigt werden. Auch künftig wird es - aufgrund der sehr unterschiedlichen Bedingungen - nicht immer in 30 Minuten erledigt sein.

Ich glaube, dass vorrangig andere Motivationen hinter dem Wunsch stehen.
Man will es, weil ...

1. ... man es kann. Motivation: Spieltrieb
2. ... man mehr Leistung als die erlaubten 600Wip nutzen will.
3. ... man sich "netzdienlich" verhalten will und der Sorge entgegentreten will, dass die paar Watt Einspeisung einen Blackout provozieren
4. ... man aus politischen / ideologischen Gründen keine kWh an die Raubritter verschenken will.
5. ... man aus psychologischen Gründen die Kontrolle braucht.

;)

**Reply by helgeerbe** [reply]
6. ... Wenn man noch einen Batteriespeicher hat, und lieber die Batterie lädt, als den Überschuss in Netz zu pusten.

**Reply by battifreund** [reply]
Das war schon in Punkt 4 enthalten. Allerdings sind 600Wip und Batterie jetzt nicht unbedingt ein Traumpaar.

**Reply by helgeerbe** [reply]
Stimmt, wo wir wieder bei 1. sind :)

**Comment by Solarteur** [comment]
7. Weil es nicht überall auf dieser Welt die Rückspeisung so handgehabt wird wie in Deutschland, darum darf/muss ich NULL einspeisen (Zitat von Ziyat T. aus dem Thread "Wechselrichter Hoymiles HM-xxxx 2,4 GhZ Nordic Protokoll" auf mikrocontroller.net)

**Comment by kpwg** [comment]
8. Weil es tatsäch Zähler gibt, welche den Bezug 1.8.0 und die Einspeisung 2.8.0 nicht trennen, sondern beide Richtungen dem Bezug zurechnen. In der Praxis bezahlt man dann für den eingespeisten Überschuss an den Netzbetreiber.

**Reply by madmartin** [reply]
Das halte ich für eine Internet-Legende.

**Reply by hartkopp** [reply]
Ist keine Legende. Diese Logos mit "Drehstromzähler always-positive" gibt es wirklich auf manchen Zählern:
https://www.energie-experten.org/news/achtung-balkon-pv-always-positive-zaehler-werten-einspeisung-als-verbrauch

**Reply by madmartin** [reply]
> Ist keine Legende. Diese Logos mit "Drehstromzähler always-positive" gibt es wirklich auf manchen Zählern: https://www.energie-experten.org/news/achtung-balkon-pv-always-positive-zaehler-werten-einspeisung-als-verbrauch

Ach komm das ist doch ein Aprilscherz den irgendwer mal losgetreten hat. Und dann schreiben viele von vielen anderen ab. Schau Dir mal das Logo  in 

https://www.energie-experten.org/fileadmin/Newsartikel/Newsartikel_07/Achtung_Balkon-PV_Always_positive_Zaehler_werten_Einspeisung_als_Verbrauch_03_Grafik_EasyMeter.jpg

an, das hat doch jemand in die Liste hineingebastelt, es hat ne ganz andere Auflösung als die anderen.

**Reply by hartkopp** [reply]
Hm, könnte sein. Danke für die Referenz. 

https://twitter.com/dk3ml/status/1578301697541443589/photo/1

Das Bild im referenzierten Twitter Post sieht aber "echter" für Dich aus, oder?

**Comment by hartkopp** [comment]
@tbnobody : Ich habe ein Linux Shell-Script für OpenWrt/RasPi/whatever geschrieben, was mit der OpenDTU WebAPI eine Nulleinspeisung mit einem Tasmota Smartmeter WLAN Interface und OpenDTU realisiert:

https://github.com/hartkopp/zerofeed

Läuft jetzt seit Tagen auf meinem OpenWrt Router - mir fehlt nur noch ein Start-Stop-Script für OpenWrt, damit es so _richtig schön_ wird.
Alles nur mit `curl`und `jq`, damit es auf jeder kleinen Linux Box läuft - zu MQTT habe ich irgenwie keinen Zugang gefunden ;-) 

Besteht ein Interesse, das Script im README.md unter "Web API Documentation" oder "Related Projects" zu verlinken?

**Reply by madmartin** [reply]
> @tbnobody : Ich habe ein Linux Shell-Script für OpenWrt/RasPi/whatever geschrieben, was mit der OpenDTU WebAPI eine Nulleinspeisung mit einem Tasmota Smartmeter WLAN Interface und OpenDTU realisiert:
> 
> https://github.com/hartkopp/zerofeed
> 
> Läuft jetzt seit Tagen auf meinem OpenWrt Router 

Cool!!! Genau mein Geschmack! Vor allem gefällt mir die Idee solche kleinen Helfer auch mal auf einem Router zu platzieren.


> - mir fehlt nur noch ein Start-Stop-Script für OpenWrt, damit es so _richtig schön_ wird. 

Hier ist die Doku dazu:  https://openwrt.org/docs/techref/initscripts


> Besteht ein Interesse, das Script im README.md unter "Web API Documentation" oder "Related Projects" zu verlinken?

Auf jeden Fall!

**Reply by martin8310** [reply]
Und jetzt nur noch etwas für NodeRED bauen und alles wäre top.

**Reply by hartkopp** [reply]
> > * mir fehlt nur noch ein Start-Stop-Script für OpenWrt, damit es so _richtig schön_ wird.
> 
> Hier ist die Doku dazu: https://openwrt.org/docs/techref/initscripts

Erledigt ;-)
https://github.com/hartkopp/zerofeed/commit/143682e1485cce235aca15bfebc2150dc2a465e3

Ich werde die Doku aus dem Commit nochmal ins README übertragen - aber nicht mehr heute.
